monkey/monkey_island/cc/resources/exploitations/manual_exploitation.py

```diff
@@ -7,6 +7,12 @@
 
 
 class ManualExploitation(flask_restful.Resource):
+    # API Spec: RPC-style endpoint? Getting manual exploitations is an action and there's no
+    # "resource", however one could argue that at some point in time, we may want to access
```

Comment:
I don't understand what qualifies a "resource"? To the API consumer there's no difference if there is a manual exploitation in the database or if it's just created on a get request. This is only used in ransomware report, maybe it should go under the report resource? Like /api/reports/manual-exploitations

Reply:
I got it slightly mixed up previously. This qualifies as a resource.